app/main/templates/index.html

```diff
@@ -3,117 +3,86 @@
+            <div class="hazen-dotted-horizontal-line py-3"></div>
+        </section>
 
-<body style="background: #FFFFFF" text-align="center">
+        <!-- Introductory Text -->
+        <section class="pb-5">
+            <div class="row">
+                <p class="p-front-page lead">
+                    <span class="hazen-within-p">HAZEN</span> is a software framework for performing automated analysis of magnetic resonance imaging (MRI)
+                    quality assurance data. Hazen consists of multiple image processing Tasks which perform
+                    quantitative analysis of MRI phantom data.
+                </p>
+                <p class="p-front-page lead">
+                    Hazen currently supports the ACR Large MRI Phantom and
+                    the MagNET Test Objects collection of phantoms.
+                </p>
+            </div>
 
-  <h1 data-text="Welcome to Hazen" >Welcome to Hazen, automated QA</h1>
-  <p style="font-size:150%">Hazen is an automated quality assurance and analytics tool for the purpose of assessing performance of MRI systems and their related hardware and software.</p>
+            <div class="hazen-dotted-horizontal-line py-3"></div>
+        </section>
 
-  <div class="container ">
-    <div class="row row-cols-1 row-cols-md-3 g-4">
-      <div class="col-sm-4">
-        <div class="card" style="width: 18rem;">
-          <img src="{{url_for('static', filename = 'uniformity.jpg') }}" class="card-img-top" alt="uniformity">
-          <div class="card-body">
-            <h5 class="card-title">Uniformity</h5>
-            <h6 class="card-subtitle mb-2 text-muted"></h6>
-            <p class="card-text"></p>
-            <div class="card-footer">
-              <a href="https://hazen.readthedocs.io/en/latest/tasks.html#uniformity" class="btn mr-2"><i class="fas fa-link"></i>Docs</a>
-              <a href="https://github.com/GSTT-CSC/hazen-web-app" class="btn"><i class="fab fa-github"></i>GitHub</a>
-            </div>
-          </div>
-        </div>
-      </div>
-      <div class="col-sm-4">
-        <div class="card" style="width: 18rem;">
-          <img src="{{url_for('static', filename = 'snr.jpg') }}" class="card-img-top">
-          <div class="card-body">
-            <h5 class="card-title">SNR</h5>
-            <h6 class="card-subtitle mb-2 text-muted"></h6>
-            <p class="card-text"></p>
-            <div class="card-footer">
-              <a href="https://hazen.readthedocs.io/en/latest/tasks.html#signal-to-noise-ratio-snr" class="btn mr-2"><i class="fas fa-link"></i>Docs</a>
-              <a href="https://github.com/GSTT-CSC/hazen-web-app" class="btn"><i class="fab fa-github"></i>GitHub</a>
+        <!-- Task List -->
+        <section class="pb-5">
+            <div class="row">
+                <h2 class="h2 pb-3">HAZEN Tasks</h2>
+                <p class="p-front-page lead">
+                    Hazen provides the following measurements of MRI test objects:
```

Comment:
are these the only tasks? do we have to specify them manually?
could we add 2 columns for the set of tasks for the ACR and MagNet phantoms?

Reply:
I omitted MR Relaxometry as it's not widely used. But I think that covers them.

To be honest, the front page design is somewhat placeholder. The front page should just be introductory and not overwhelm new users with info. But I do agree it would be nice to convey the tasks in a better way, perhaps with a 2 column option like the one you describe.

Can we revise this further along once we've been through web-app testing?